gr-utils/python/modtool/core/add.py

```diff
@@ -143,38 +143,41 @@ def run(self):
 
     def _run_cc_qa(self):
         " Add C++ QA files for 3.7 API if intructed from _run_lib"
-        fname_qa_h  = 'qa_{}.h'.format(self.info['blockname'])
-        fname_qa_cc = 'qa_{}.cc'.format(self.info['blockname'])
+        blockname_=self.info['blockname']
+        fname_qa_h  = f'qa_{blockname_}.h'
+        fname_qa_cc = f'qa_{blockname_}.cc'
         self._write_tpl('qa_cpp', 'lib', fname_qa_cc)
         self._write_tpl('qa_h',   'lib', fname_qa_h)
+        modname_= self.info['modname']
         if self.skip_cmakefiles:
             return
         try:
             append_re_line_sequence(self._file['cmlib'],
-                                    r'list\(APPEND test_{}_sources.*\n'.format(self.info['modname']),
-                                    'qa_{}.cc'.format(self.info['blockname']))
+                                  fr'list\(APPEND test_{modname_}_sources.*\n',
+                                    f'qa_{blockname_}.cc')
             append_re_line_sequence(self._file['qalib'],
                                     '#include.*\n',
-                                    '#include "{}"'.format(fname_qa_h))
+                                    f'#include "{fname_qa_h}"')
             append_re_line_sequence(self._file['qalib'],
                                     '(addTest.*suite.*\n|new CppUnit.*TestSuite.*\n)',
-                                    '  s->addTest(gr::{}::qa_{}::suite());'.format(self.info['modname'],
-                                                                                   self.info['blockname'])
+                                    f'  s->addTest(gr::{modname_}::qa_{blockname_}::suite());'
                                     )
             self.scm.mark_files_updated((self._file['qalib'],))
         except IOError:
             logger.warning("Can't add C++ QA files.")
 
     def _run_cc_qa_boostutf(self):
         " Add C++ QA files for 3.8 API if intructed from _run_lib"
-        fname_qa_cc = 'qa_{}.cc'.format(self.info['blockname'])
+        blockk_=self.info['blockname']
```

Comment:
looks a bit like you had a typo there; also, you called the same thing `blockname` above, I think using the same name here would be advantageous.